src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ScoreSystem.java

```diff
@@ -97,12 +97,7 @@ private void checkAndResetGameOnScore(ActivateEvent event, EntityRef entity) {
     }
 
     private void sendGameOverEventToClients(String winningTeam) {
-        if (entityManager.getCountOfEntitiesWith(ClientComponent.class) != 0) {
-            Iterable<EntityRef> clients = entityManager.getEntitiesWith(ClientComponent.class);
-            for (EntityRef client : clients) {
-                client.send(new GameOverEvent(winningTeam, blackScore, redScore));
-            }
-        }
+        sendEventToClients(new GameOverEvent(winningTeam, blackScore, redScore));
```

Comment:
with this, I think we even can drop this method and inline this in `checkAndResetGameOnScore`